bucket/maven.json

```diff
@@ -6,9 +6,6 @@
     "hash": "90e9f0d700743516d1f610fcd63a5d49751d0743db713909f9ef933747a38b8a",
     "extract_dir": "apache-maven-3.5.2",
     "env_add_path": "bin",
-    "env_set": {
-        "M2_HOME": "$dir"
```

Comment:
`M2_HOME` was [deprecated](https://maven.apache.org/docs/3.5.0-alpha-1/release-notes.html) in 3.5.0-alpha and is no longer used by the `current` version.

> Based on problems in using `M2_HOME` related to different Maven versions installed and to simplify things, the usage of `M2_HOME` has been removed and is not supported any more MNG-5823, MNG-5836, MNG-5607.

With that being said, if a scoop user decided to run `scoop install maven@3.3.9` they would prefer to have `M2_HOME`. However, since version support isn't widely publicized, it's probably safe to go ahead and remove it. 👍